src/Core/NuGet/Core.NuGet.csproj

```diff
@@ -2,7 +2,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildProjectDirectory), 'Common.props'))\Common.props" />
 
   <PropertyGroup>
-    <TargetFrameworks>net45</TargetFrameworks>
+    <TargetFrameworks>netstandard1.3</TargetFrameworks>
```

Comment:
Is it supposed to be only net1.3? Will it install over 4.0/4.5/4.6?
I see that other projects you were changing are now targeting several net versions at the same time.